### DISCUSSION TITLE
[AGS] How can I remove/disable some functions?

### DISCUSSION THREAD

**Discussion by bushaev-denis** [end-4/dots-hyprland#384] [General]
I don't want to use all things in left sidebar, also i don't need stats in system bar, how can i remove this functionality?
Also i couldn't find how can i change weather location.

**Comment by bushaev-denis** [comment]
Ok, i just noticed about arg directory, I disabled sidebar in JS.

**Comment by end-4** [comment]
to change system stats widget edit `~/.config/ags/modules/bar/normal/music.js` <- i should rename this...
to change weather location, add the following to the `userConfigOptions` object in `~/.config/ags/user_options.js`
```js
    'weather': {
        'city': "bangkok",
    },
```

**Reply by MorningMC** [reply]
I'm using illogical-impulse right now, there is neither .config/ags folder, nor js files in .config/illogical-impulse folder how can i disable left sidebar?

**Reply by end-4** [reply]
see ~/.config/quickshell/ii/shell.qml
